packages/back-end/src/events/handlers/slack/slackEventHandler.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  NotificationEventName,
+  NotificationEventPayload,
+} from "../../base-types";
+import {
+  FeatureCreatedNotificationEvent,
+  FeatureDeletedNotificationEvent,
+  FeatureUpdatedNotificationEvent,
+} from "../../base-events";
+import { FeatureUpdatedNotificationHandler } from "../../notifiers/FeatureUpdatedNotifier";
+
+/**
+ * handle Slack events. Can be handled individually or with a common handler, depending on needs.
+ * @param payload
+ */
+export const slackEventHandler = async (
```

Comment:
While this handler is included here, it is not in use and has been commented out. I wanted to provide another example for how we can handle events on a per-type basis. We can change and include this file once we start adding support for Slack.